<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesión</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-radius:8px</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagenes.qrc">:/boca/Escudo_del_Club_Atlético_Boca_Juniors.svg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio de sesión</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ingrese su usuario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClave">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Ingrese su contraseña</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcceder">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 0);
color: rgb(0, 0, 186);</string>
   </property>
   <property name="text">
    <string>Acceder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblError">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 11, 3);</string>
   </property>
   <property name="text">
    <string>Mensaje</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
